sycl/include/CL/sycl/INTEL/esimd/esimd_view.hpp

```diff
@@ -89,21 +89,6 @@ template <typename BaseTy, typename RegionTy> class simd_view {
     return getTopRegion(M_region).M_offset_y;
   }
 
-  template <int Dim = 0> static constexpr int getSize() {
-    static_assert(Dim <= is2D(), "region is not two-dimensional");
-    return (Dim == 0) ? getSizeX() : getSizeY();
-  }
-
-  template <int Dim = 0> static constexpr int getStride() {
-    static_assert(Dim <= is2D(), "region is not two-dimensional");
-    return (Dim == 0) ? getStrideX() : getStrideY();
-  }
-
-  template <int Dim = 0> constexpr uint16_t getOffset() const {
```

Comment:
Also need to update the reference in sycl/test/basic_tests/esimd/simd.cpp, to replace getOffset<0>()/getOffset<1>() with getOffsetX()/getOffsetY().